src/tools/kvstore_tool.cc

```diff
@@ -38,6 +33,17 @@ StoreTool::StoreTool(const string& type, const string& path, bool need_open_db)
   }
 }
 
+void StoreTool::start_bluestore(const string& path, bool need_open_db)
```

Comment:
nit, might want to rename this method to `load_bluestore()` or `open_bluestore()`.

Reply:
Done.